packages/react/src/Delay.tsx

```diff
@@ -1,17 +1,21 @@
-import { type PropsWithChildren, useContext, useState } from 'react'
+import { type PropsWithChildren, type ReactNode, useContext, useState } from 'react'
 import { DelayDefaultPropsContext } from './contexts'
 import { useTimeout } from './hooks'
 
 export interface DelayProps extends PropsWithChildren {
   ms?: number
+  /**
+   * @experimental This is experimental feature.
+   */
+  fallback?: ReactNode
 }
 
-export const Delay = ({ ms, children }: DelayProps) => {
+export const Delay = ({ ms, children, fallback }: DelayProps) => {
   const defaultProps = useContext(DelayDefaultPropsContext)
   const delayMs = ms ?? defaultProps.ms ?? 0
   const [isDelayed, setIsDelayed] = useState(delayMs === 0)
   useTimeout(() => setIsDelayed(true), delayMs)
```

Comment:
The intuitive nuance of isDelayed is that it has a stronger sense of "was it delayed?" (past tense) and the actual usage is stronger "did you wait for all the delayed time?", so why not change it to isDelaying and swap the true false?

> 한국말로, 지연되었는지?

```suggestion
  const [isDelaying, setIsDelaying] = useState(delayMs !== 0)
  useTimeout(() => setIsDelaying(false), delayMs)
```

It may not make a huge difference in readability, but let me know what you think!

Reply:
Cool, I renamed it as isDelaying in [aafcfc6](https://github.com/suspensive/react/pull/555/commits/aafcfc6f0f560647a18918b0dc74d8068bece787)